python/print_timestamp.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, tstamp_format='', count_packets=False):
 
     def handle_msg(self, msg_pmt):
         if self.tstamp_format:
-            timestamp = datetime.datetime.utcnow()
+            timestamp = datetime.datetime.now(datetime.timezone.utc)
+            timestamp = timestamp.replace(tzinfo=None)
```

Comment:
Here I think it's fine to keep the UTC timezone awareness. The timestamp is just formatted with `strftime()`, so I don't think the output will change regardless of whether there is a timezone or not, but having the timezone allows the `tstamp_format` string to use `%:z` to get the timezone printed, if the user wishes to.